### PR TITLE
Buy view

### PR DESCRIPTION
**Pull request by Z-Ramezani**

### CODE REVIEW

File: Enigma/buy/serializers.py [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+
+class consumerSerializer(serializers.ModelSerializer):
+    consumer_cont = serializers.RelatedField(many=True, read_only=True)
+    class Meta:
+        model = consumer
         fields = "__all__"
 
 
 class buySerializer(serializers.ModelSerializer):
     buyers = buyerSerializer(many=True, read_only=True)
```
Comment (Shafizadegan) [inline Enigma/buy/serializers.py]: it is correct

File: Enigma/buy/serializers.py [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
-
+    consumers = consumerSerializer(many = True, read_only= True)
```
Comment (Shafizadegan) [inline Enigma/buy/serializers.py]: ok

File: Enigma/buy/views.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 from buy.serializers import BuySerializer
 
 
-class (APIView):
+class GetGroupBuys(APIView):
 
     def post(self, request):
         perch = buy.objects.filter(
```
Comment (Shafizadegan) [inline Enigma/buy/views.py:11]: serializer is correct

**Comment by Z-Ramezani** [comment]
conflicts!!!!

**Comment by Z-Ramezani** [comment]
!!!!

File: Enigma/buy/views.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 from buy.serializers import BuySerializer
 
 
-class (APIView):
+class GetGroupBuys(APIView):
```
Comment (Shafizadegan) [inline Enigma/buy/views.py:8]: correct

**Comment by Shafizadegan** [comment]
it is ok
thanks

File: Enigma/Group/migrations/0001_initial.py [hunk 5]
```diff
@@ -27,7 +27,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('groupID', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='group', to='Group.group')),
-                ('userID', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='member', to=settings.AUTH_USER_MODEL)),
+                ('userID', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='member', to=settings.AUTH_USER_MODEL)),
```
Comment (fatemeshafiee) [inline Enigma/Group/migrations/0001_initial.py:30]: Why null?

**Review by fatemeshafiee — APPROVED** [review]
Seems, ok

File: Enigma/Group/migrations/0001_initial.py (hunk 5 above)
Comment (Z-Ramezani) [reply Enigma/Group/migrations/0001_initial.py:30]: for serializer.

**Pull request closed without merging**